data/genesis/imap/errorlimit.rb

```diff
@@ -51,7 +51,7 @@
   
   
   # set the default IMAP error counter
-  RunCommandOnMailbox.new('zmlocalconfig -e imap_max_consecutive_error=0'),
+  RunCommandOnMailbox.new('zmlocalconfig -e imap_max_consecutive_error=5'),
```

Comment:
@RohitVyawahare I'm curious how this (and other `zmlocalconfig` updates) will work (or even be applicable) when we are testing a system with a pool of mailbox servers to which consecutive API calls may be routed to any mailbox server?

Reply:
I agree with you, not sure yet how are we going to handle localconfig for HA environment since we are considering a pool of mailbox servers working for a common cause, setting some configuration for a single mailbox wouldn't be valid use case I think. 

I have just updated this testcase to execute on the mailbox because imap_max_consecutive_error needs to be set on mailbox with current architecture so that it atleast works fine with current configuration.

Reply:
FWIW, previously there was work done for "always on" / 9.x to migrate things from localconfig to LDAP - for example https://bugzilla.zimbra.com/show_bug.cgi?id=96883

Reply:
Thank you @plobbes